bootstrap.yml

```diff
@@ -581,7 +626,6 @@ packages:
         - host-pkg-config
       regenerate:
         - args: ['./autogen.sh']
-          environ:
             'NOCONFIGURE': 'yes'
```

Comment:
This seems to be an oversight? (Without `environ`, the `NOCONFIGURE` line does nothing.)